src/modules/core/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -19,23 +18,19 @@ const BreadCrumb = ({ elements }: Props) => {
     <div className={styles.crumbContainer}>
       {elements.map((crumb, i) => (
         <div className={styles.element} key={`breadCrumb_${crumb}`}>
-          {elements.length > 1 && i < elements.length - 1 ? (
-            <>
-              <span className={styles.breadCrumble}>
-                <FormattedMessage {...Object.values(crumb)[0]} />
-              </span>
-              <span className={styles.arrow}>&gt;</span>
-            </>
-          ) : null}
           <>
-            {i === elements.length - 1 || elements.length === 1 ? (
+            {elements.length > 1 && i < elements.length - 1 ? (
               <>
-                <b className={styles.breadCrumble}>
-                  <FormattedMessage {...Object.values(crumb)[0]} />
-                </b>
+                <span className={styles.breadCrumble}>{crumb}</span>
+                <span className={styles.arrow}>&gt;</span>
               </>
             ) : null}
           </>
+          <>
+            {i === elements.length - 1 || elements.length === 1 ? (
+              <b className={styles.breadCrumble}>{crumb}</b>
+            ) : null}
```

Comment:
I think this can be simplified a little bit. Since the falsy expression in each of these ternaries is `null`, these don't need to be ternaries.

Also, those surrounding fragments aren't necessary 🙂

Something along the lines of:

```javascript
<div className={styles.element} key={`breadCrumb_${crumb}`}>
  {elements.length > 1 && i < elements.length - 1 && (
    <>
      <span className={styles.breadCrumble}>{crumb}</span>
      <span className={styles.arrow}>&gt;</span>
    </>
  )}
  {(i === elements.length - 1 || elements.length === 1) && (
    <b className={styles.breadCrumble}>{crumb}</b>
  )}
</div>
```